xicam/plugins/ingestorplugin.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
great comments here :laughing: it explains why we need this class here for it to be discovered and loaded by the plugin manager